ledger/simulation/trace.go

```diff
@@ -103,20 +104,41 @@ func (eo ResultEvalOverrides) LogicEvalConstants() logic.EvalConstants {
 	return logicEvalConstants
 }
 
+// ExecTraceConfig gathers all execution trace related configs for simulation result
+type ExecTraceConfig struct {
+	Enable bool
+}
+
```

Comment:
It feels like a lot of indirection for us to define this by hand, while we define the same structure as a `model`, as then write a bunch of copy routines in each direction.

Reply:
Yea, the design concern here is not to make trace-related configuration goes sporadic across request and response file, but the cost is to add 2 more copying routines from/to model objects...

Reply:
The thinking is to somehow be similar to `EvalOverride` to gather related overriding configs together, but it might apply differently here. Wonder if @jasonpaulos could also weigh in here?

Reply:
oh boy I was being dumb, see [remove exec trace config copying routines, use codec reflect marshalling](https://github.com/algorand/go-algorand/pull/5360/commits/2610b29b7b3c33b7708a3b0b4d761bf1546b6b89), we can use `codec` in `simulation.ExecTraceConfig` to get away from the non-necessary copying routine.